_includes/course-page.html

```diff
@@ -0,0 +1,47 @@
+
+<div id="hero-js" class="hero py-11">
```

Comment:
Lets be able to pass in the id here so we could potentially have different pictures